02/app.js

```diff
@@ -9,12 +9,21 @@ class App extends React.Component {
         usersList: [],
     }
 
+    updateUsersList = evt => {
+      evt.preventDefault();
+      const usersList = this.state.usersList.map(item => item);
```

Comment:
Moim zdaniem ładniej jest zrobić to tak: `const usersList = [...this.state.usersList]`